app/app/services/argyle_service.rb

```diff
@@ -40,4 +42,8 @@ def initialize(environment, api_key_id = nil, api_key_secret = nil)
   def items(query = nil)
     @http.get("items", { q: query }).body
   end
+
+  def create_user
+    @http.post(USERS_ENDPOINT).body
```

Comment:
This could just be

```suggestion
    @http.post("/users").body
```

since we configure the HTTP client with the base_url on line 27. This would be preferable because it's fewer URLs to change based on the Argyle environment.